scripts/dev/format.sh

```diff
@@ -1,87 +1,119 @@
-#!/bin/sh
+#!/bin/bash
 
 # SPDX-License-Identifier: Apache-2.0
 #
 # Copyright (C) 2021 Micron Technology, Inc. All rights reserved.
 
-if ! type clang-format > /dev/null 2>&1; then
-    >&2 echo "clang-format not found"
-    exit 1
-fi
+CMD=${0##*/}
+
+CLANG_FORMAT_VERSION=13
```

Comment:
I thought we were using something from 15 or 14?

Reply:
I don't see the value.  This config was based on 13 and should work seamlessly with 14, 15 and 16.  I will verify.

Reply:
You should know by now that Tristan likes to live on the bleeding edge :-)
FWIW, I verified this morning that fc36 is on kernel 6 something...

Reply:
Aren't you using https://clang.llvm.org/docs/ClangFormatStyleOptions.html#spacebeforeparensoptions which is only available in 14?

Reply:
I'm using `SpaceBeforeParens: ControlStatementsExceptControlMacros`

Reply:
Ok. Sounds good.